RecoEgamma/PhotonIdentification/python/PhotonIDValueMapProducer_cff.py

```diff
@@ -0,0 +1,9 @@
+from RecoEgamma.PhotonIdentification.photonIDValueMapProducer_cfi import *
+
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(photonIDValueMapProducer, 
+    esReducedRecHitCollection = cms.InputTag(""),
+    esReducedRecHitCollectionMiniAOD = cms.InputTag(""),
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
Ok! So that means just `esReducedRecHitCollection = ""`?

Reply:
yes, thanks